av1an-core/src/scene_detect.rs

```diff
@@ -242,26 +242,16 @@ fn build_decoder(
           .arg("y4m")
           .arg(path)
           .arg("-")
+          .env("AV1AN_PERFORM_SCENE_DETECTION", "true")
           .stdin(Stdio::null())
           .stdout(Stdio::piped())
           .stderr(Stdio::null());
         // Append vspipe python arguments to the environment if there are any
         for arg in vspipe_args {
           command.args(["-a", &arg]);
         }
-        let vspipe = command.spawn()?.stdout.unwrap();
-        Decoder::Y4m(y4m::Decoder::new(
-          Command::new("ffmpeg")
-            .stdin(vspipe)
-            .args(["-i", "pipe:", "-f", "yuv4mpegpipe", "-strict", "-1"])
-            .args(filters)
-            .arg("-")
-            .stdout(Stdio::piped())
-            .stderr(Stdio::null())
-            .spawn()?
-            .stdout
-            .unwrap(),
-        )?)
+
+        Decoder::Y4m(y4m::Decoder::new(command.spawn()?.stdout.unwrap())?)
```

Comment:
Oh. Yeah, we don't need ffmpeg at all here because we can do the resizing inside of Vapoursynth now. Nice.